kaniko/kaniko.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: tekton.dev/v1alpha1
```

Comment:
Starting to think through the testing story here, WDYT about also including a valid TaskRun? You can inline the resources. At a minimum (you don't have to do it in this PR), we could create all of the supplied Runs and make sure they succeed.

Reply:
I had thought about it. That gets tricky when the Task requires auth to successfully push the image. It's the same reason the build-templates repo never included actual running builds... 😒

Reply:
Hmm, maybe we'll have to come up with some conventions on what auth we'll provide as part of the test. Thanks for the explanation!